### ISSUE TITLE
pytorch 강좌 수강

### ISSUE THREAD

**Issue by heehehe** [heehehe/pruning-decorrelation#4]
pytorch zero to all 강의 수강
https://www.edwith.org/boostcourse-dl-pytorch
https://deeplearningzerotoall.github.io/season2/lec_pytorch.html

### Part 1 - Basic ML (10/10-12)
- [x] Lab-01-1 Tensor Manipulation 1 (10/10)
- [x] Lab-01-2 Tensor Manipulation 2 (10/10)
- [x] Lab-02 Linear regression (10/10)
- [x] Lab-03 Deeper Look at GD (10/11)
- [x] Lab-04-1 Multivariable Linear regression (10/11)
- [x] Lab-04-2 Loading Data (10/11)
- [x] Lab-05 Logistic Regression (10/11)
- [x] Lab-06 Softmax Classification (10/11)
- [x] Lab-07-1 Tips (10/12)
- [x] Lab-07-2 MNIST Introduction (10/12)

### Part 2 - DNN (10/13-14)
- [x] Lab-08-1 Perceptron (10/13)
- [x] Lab-08-2 Multi Layer Perceptron (10/13)
- [x] Lab-09-1 ReLU (10/13)
- [x] Lab-09-2 Weight initialization (10/14)
- [x] Lab-09-3 Dropout (10/14)
- [x] Lab-09-4 Batch Normalization (10/14)

### Part 3 - CNN (10/15-18)
- [x] Lab-10-1 Convolution (10/15)
- [x] Lab-10-2 mnist cnn (10/18)
- [x] Lab-10-3 visdom (10/18)
- [x] Lab-10-4-1 ImageFolder1 (10/18)
- [x] Lab-10-4-2 ImageFolder2 (10/18)
- [x] Lab-10-5 Advance CNN(VGG) (10/18)
- [x] Lab-10-6-1 Advanced CNN(RESNET-1) (10/12+18)
- [x] Lab-10-6-2 Advanced CNN(RESNET-2) (10/18)
- [x] Lab-10-7 Next step of CNN (10/18)

### Part 4 - RNN (11월)
- [x] Lab-11-0 RNN intro
- [x] Lab-11-1 RNN basics
- [x] Lab-11-2 RNN hihello and charseq
- [x] Lab-11-3 Long sequence
- [x] Lab-11-4 RNN timeseries
- [x] Lab-11-5 RNN seq2seq
- [x] Lab-11-6 PackedSequence
